content/security-humanops.rst

```diff
@@ -0,0 +1,231 @@
+Other email tips
+----------------
+
+Add the Pro:Atria Email Signature Template in your emails.
+
+Set your email settings to prevent HTML/images loading and to block remote content.  
+
+Utilize a blacklist of known spam senders.
+
+
+Use of group emails
+-------------------
+
+Do not add team members that are still on trial to the group until such addition is verified by the team lead.
+
+Do not use team group emails for non SFTPPlus purposes like personal and industry newsletters, announcements, and so on.
+
+There may be some cases where we need to use the group email for related services. When using team group emails for SFTPPlus testing or to open SFTPPlus accounts please notify the group email first that the testing / account is legitimate.  That way, when a test is sent to a team account it is not immediately assumed that it is legitimate. 
+
+
+Phishing and Social Engineering
+===============================
+
+If a phishing attack is encountered or you suspect a phishing attack is encountered, please notify the team.  
+
+Phishing is utilized as some form of information gathering (eg. enumerating) taking place.
+
+If encountering potential SE (social engineering) attacks, please notify the team lead.
+
+
+Passwords
+=========
+
+At some point you will need to use services that are reliant on passwords.
+
+If possible, use non-password options.
+
+Use password management tools (ie LastPass, 1Password, etc) and ensure to enable MFA for these tools.
+
+Do not reuse passwords from your personal accounts with SFTPPlus accounts.
+
+Do not send passwords via cleartext. Or even better, do not continue to use services that send passwords via cleartext.
+
+When changing passwords do not use common password mutations like changing a character at the end.
+
+Do not allow opportunities for password profiling, like using company products in your password.
+
+
+Internet facing information
+===========================
+
+Consider what services are facing the Internet.
+
+Consider what information is facing the Internet.
+
+When posting reviews or comments, are there certain details that need to be taken out to sanitize the review or comment?  
+
+Is this public information going to help with someone's enumeration activities? Are there sensitive files, passwords, private key info publicly facing? 
+
+
+Customers
+=========
+
+We work with admins that are already aware of security practices - like not leaving sensitive information in logs.
+
+However we work with a number of customer bases and budgets.
+
+
+Olark
+-----
+
+We use Olark web chat. If talking to a customer, please verify who they are and their installation.  The other alternative is to move the discussion to email.
+
+
+Improving Documentation
+-----------------------
+
+If there is a customer having trouble securing their setup and they have read the documentation, ask what they are trying to do.  Then detect what portion of the documentation can be improved and create a ticket for this.
```

Comment:
I think that this should go here http://styleguide.chevah.com/documentation.html